flight/Cargo.toml

```diff
@@ -0,0 +1,24 @@
+[package]
+name = "flight"
+version.workspace = true
+edition.workspace = true
+authors.workspace = true
+license.workspace = true
+repository.workspace = true
+
+[dependencies]
+anyhow.workspace = true
+cid.workspace = true
+serde.workspace = true
+serde_bytes.workspace = true
+arrow = "9.0.0"
```

Comment:
also, just noticed this. It should be `arrow.workspace=true` so all crates are using the same arrow version (i.e. 52)